spec/features/cancel_spec.rb

```diff
@@ -1,13 +1,9 @@
 describe "Canceling a request" do
-  let (:client_data) { FactoryGirl.create(:ncr_work_order) }
-  let (:proposal) { FactoryGirl.create(:proposal, :with_approver, client_data_type: "Ncr::WorkOrder", client_data_id: client_data.id) }
-  let (:user) { FactoryGirl.create(:user, id: 123456) }
-
-  before do
-    login_as(proposal.requester)
-  end
+  let!(:client_data) { FactoryGirl.create(:ncr_work_order, :with_approver) }
+  let!(:proposal) { client_data.proposal }
 
   it "shows a cancel link for the requester" do
+    login_as(proposal.requester)
```

Comment:
The reason is that we need different users logged in for different tests (an approver vs. the requester, etc.) I had set up `contexts` around who was logged in and was doing this in `before` blocks, but it made the test grouping somewhat confusing. I think this is a case where being DRY isn't actually helpful.